tools/linconym.py

```diff
@@ -346,18 +346,18 @@ def get_word_path(self, position: tuple[int]) -> list[str]:
 
         word_path: list[str] = []
         for i in range(len(position)):
-            previous_pos: tuple[int] = position[:(i+1)]
+            previous_pos: str = position[:(i+1)]
```

Comment:
Ici pour moi il faut d'abord faire un split sur la position pour se ramener à quelque chose de semblable à ce que l'on a avec les tuples. Sinon on itère sur les caractères de la chaine de caractères.